tests/queries/0_stateless/02801_backup_native_copy.reference

```diff
@@ -1,4 +1,4 @@
-BACKUP TABLE data TO S3(s3_conn, \'backups/default/data_native_copy\') SETTINGS allow_s3_native_copy = 1	1
-BACKUP TABLE data TO S3(s3_conn, \'backups/default/data_no_native_copy\') SETTINGS allow_s3_native_copy = 0	0
-RESTORE TABLE data AS data_native_copy FROM S3(s3_conn, \'backups/default/data_native_copy\') SETTINGS allow_s3_native_copy = 1	1
-RESTORE TABLE data AS data_no_native_copy FROM S3(s3_conn, \'backups/default/data_no_native_copy\') SETTINGS allow_s3_native_copy = 0	0
+BACKUP TABLE data TO S3(s3_conn, \'backups/default/data_native_copy\') SETTINGS allow_s3_native_copy=true	1
```

Comment:
Is this related to improving hiding secrets?

Reply:
I changed the hiding algorithm a bit so now it returns the same query as it was if there was nothing to hide. I mean it used to replace `true` with `1`, and now it doesn't do that anymore.